pygfx/cameras/_orthographic.py

```diff
@@ -13,7 +13,7 @@ class OrthographicCamera(PerspectiveCamera):
         may be wider if the viewport is relatively wide.
     height: float
         The (minimum) height of the view-cube. The actual view
-        may be heigher if the viewport is relatively heigh.
+        may be higher if the viewport is relatively height.
```

Comment:
Wrong again, should be:

```suggestion
        may be taller if the viewport is relatively high.
```